gun-violence-dashboard/package.json

```diff
@@ -46,4 +46,4 @@
       "last 1 safari version"
     ]
   }
```

Comment:
I'm not able to run this code without npm installing a bunch of libraries myself. I'm getting a bunch of missing package issues. Maybe try removing node_modules and npm install to see if it works on your machine from scratch.